java/build.gradle

```diff
@@ -6,6 +6,7 @@ buildscript {
 
 plugins {
     id 'de.fuerstenau.buildconfig' version '1.1.8'
+	id 'checkstyle'
```

Comment:
It seems the formatting is wrong

Reply:
Thanks.
Yes, indeed, I've used a tab here rather than spaces (conforming with surroundings). 🤦 

Kind of ironic given I'm currently working up my next PR to convert tabs to spaces in all Java files! 😆

Reply:
https://github.com/ably/ably-java/pull/609/commits/8f4ea5a68ce307b1589f90a3c260d097ed0d20fa